### PR TITLE
[PDMO-67]-Redesign Patient Screen

### PR DESCRIPTION
**Pull request by Farhanisto**
## JIRA TICKET NAME:

[PDMO-67 Redesign Patient Screen](https://issues.openmrs.org/browse/PDMO-67)

## SUMMARY:
-Change the display of breadcrumb name from "patient search"
-Remove the accordion and display details on a single page
-Addresses should look more like the identifiers by displaying
them in a smaller card.

### CODE REVIEW

**Comment by jacksono** [comment]
Looks good, some general comments:
-where there is 'e' to mean 'event', its better to use event just to make it clear what the e means.
-there are a number of un necessary new lines, go through and remove those

**Comment by dkayiwa** [comment]
Can you look into these merge conflicts?

**Comment by Farhanisto** [comment]
Merge Conflict Solved @dkayiwa

**Comment by dkayiwa** [comment]
Now look for reviewers.

**Comment by dkayiwa** [comment]
As you look for reviewers, you again have merge conflicts.

**Comment by dkayiwa** [comment]
You also need to squash the commits into one.

**Comment by Ewanjiru** [comment]
@Farhanisto  do squash your commits

**Comment by Farhanisto** [comment]
@dkayiwa  @Ewanjiru i have squashed and resolved merge conflicts.

**Comment by cnagadya** [comment]
looks good. Though there are some two redundant new lines and I notice @jacksono had also mentioned that earlier

File: app/js/components/subAddress/addressForm.jsx [hunk 1]
```diff
@@ -265,9 +283,8 @@ export default class AddressForm extends React.Component {
           <div>
             {this.state.secondryAction === 'new' ?
               <div className="form-group">
-                <div className="col-sm-offset-4 col-sm-4">
-                  <button type="button" name="add" onClick={(e) => this.addClick(e, this.state.uuid)}
-                    className="btn btn-success form-control">Add</button>
+                <div >
+                  <FontAwesome className="fa fa-plus-square-o add-btn" onClick={(e) => this.addClick(e, this.state.uuid)} aria-hidden="true" />
```
Comment (Ewanjiru) [inline app/js/components/subAddress/addressForm.jsx]: this line seems to be too long more than 80 characters.

File: app/js/components/subAddress/addressForm.jsx [hunk 2]
```diff
@@ -184,6 +188,20 @@ export default class AddressForm extends React.Component {
...
           </div>
         </div>
+        <div className="preffered">
+                      {this.state.address.preferred ?
+                          <span className="badge badge-info">Preferred</span>
+                        :
+                        <span className="badge badge-no-color">&nbsp;</span>
+                      }
+                      {(this.state.action === "display" && this.state.secondryAction === "") ?       
+                                <FontAwesome onClick={(e) => this.editClick(e, this.state.uuid)} className="fa fa-pencil-square-o"/>
```
Comment (Ewanjiru) [inline app/js/components/subAddress/addressForm.jsx]: This line  seems to be more than 80 characters

File: app/js/components/subAddress/addressForm.jsx (hunk 1 above)
Comment (Ewanjiru) [inline app/js/components/subAddress/addressForm.jsx]: Check on this line too which seems too long

File: app/js/components/subAddress/addressForm.jsx (hunk 2 above)
Comment (Ewanjiru) [reply app/js/components/subAddress/addressForm.jsx]: Also check on variable naming, preferably e should be event

**Comment by Farhanisto** [comment]
@cnagadya @jacksono @Ewanjiru I have made the requested changes.Thanks

**Pull request merged**